pom.xml

```diff
@@ -135,6 +135,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
Also, might want to add a comment similar to kiwi & kiwi-test at top of `dependencies` section on why some things are included when otherwise they would be included transitively

Reply:
done